README.md

```diff
@@ -30,14 +30,18 @@ const validatorResult: ValidatorResult = validator.validate(tokenPairsResponse,
 
 - All token amounts are sent in amounts of the smallest level of precision (base units). (e.g if a token has 18 decimal places, selling 1 token would show up as selling `'1000000000000000000'` units by this API).
 - All addresses are sent as lower-case (non-checksummed) Ethereum addresses with the `0x` prefix.
-- `CLOSED` state means the order has been entirely filled/canceled.
 
 ## Endpoints
 
 ### GET /v0/token_pairs
 
 Retrieves a list of available token pairs and the information required to trade them.
 
+#### Parameters
+
+* token [string]: returns token pairs where tokenA or tokenB is token address
+* tokenA=&tokenB [string]: returns token pairs that contain tokenA and tokenB (in any order)
```

Comment:
@abandeali1 Couldn't we just have `tokenA=&tokenB` and make each one optional? Then we don't need `token`.